datafusion-physical-expr/src/datetime_expressions.rs

```diff
@@ -351,6 +351,8 @@ pub fn date_part(args: &[ColumnarValue]) -> Result<ColumnarValue> {
     let arr = match date_part.to_lowercase().as_str() {
         "hour" => extract_date_part!(array, temporal::hour),
         "year" => extract_date_part!(array, temporal::year),
+        "minute" => extract_date_part!(array, temporal::minute),
```

Comment:
Do we need to support `month` and other fields?
As described in the doc https://www.postgresql.org/docs/9.1/functions-datetime.html#FUNCTIONS-DATETIME-EXTRACT, There are many files needed to support.
I think we can support the kernel following this https://github.com/apache/arrow-rs/pull/1376/files in the arrow-rs before next release of arrow-rs.
I think we can implement the fields which are most used.

Reply:
Thanks, i will a 'mount 'API  in arrow-rs

Reply:
PR is https://github.com/apache/arrow-rs/pull/1388 for anyone who cases